templates/nav.html

```diff
@@ -15,9 +15,28 @@
     <div id="navbarBasicExample" class="navbar-menu">
       <div class="navbar-end">
         <div class="buttons">
-          <a class="button is-dark" href="{{ get_url(path="@/news/_index.md") }}">
-            News
-          </a>
+          {# content/_index.md doesn't actually exist, but we're using it to refer to the root section #}
+          {% set index = get_section(path="_index.md") %}
+          {% for path in index.subsections %}
```

Comment:
This is just a stylistic choice on my end, but template directives should introduce a new indent level.